packages/runtime/src/state/modules/api-resources.ts

```diff
@@ -68,6 +94,7 @@ export function getSerializedState(state: State): SerializedState {
     Array.from(resources.entries()).forEach(([compositeId, value]) => {
       if (value != null) {
         const [id, locale] = parseCompositeId(compositeId)
+        console.assert(locale == null || isLocalizableAPIResourceType(resourceType))
```

Comment:
Is this a stray console log or was the intent to keep it around?

We probably need to introduce a proper logger for our packages at some point...

Reply:
> Is this a stray console log or was the intent to keep it around?

It's intentional, it's a console **assert**, it will write an error to the console if the precondition fails: https://developer.mozilla.org/en-US/docs/Web/API/console/assert_static. I should add a message to it that allows us to identify the problematic resource, though.

> We probably need to introduce a proper logger for our packages at some point

Yes.